app/main/views/marketplace.py

```diff
@@ -145,9 +146,15 @@ def get_brief_by_id(framework_slug, brief_id):
         brief=brief,
         brief_responses=brief_responses,
         content=brief_content,
+        show_pdf_link=current_app.config.get('FEATURE_FLAGS_BRIEF_PDF', False)
     )
 
 
+@main.route('/<framework_slug>/opportunities/brief_<brief_id>.pdf')
```

Comment:
As discussed, we haven't used the word "brief" publicly in the frontend yet (AFAIK), so we should check the URL.